docs/yaml/elementary/str.yml

```diff
@@ -312,6 +314,13 @@ methods:
     '3.6'.version_compare('>=3', '<4.0') == true
     ```
 
+    *Since 1.8.1* multiple versions can be compared using comma-separated string:
```

Comment:
Also in general the next version shall be major.minor, which means 1.9.0, we do not backport new features into bugfix (major.minor.patch) releases.